### ISSUE TITLE
Wrong NeRF and surface

### ISSUE THREAD

**Issue by kafai7777** [google/nerfactor#30]
Hi, xiuming,thanks for your outstanding work and resources!

When I run the code with the scripts, settings and the your data provided in google drive, I got wrong result.

![Git1](https://user-images.githubusercontent.com/119067866/204031592-c9a5aa17-1bfd-4202-8c71-fbdeb37c778b.png)
![git2](https://user-images.githubusercontent.com/119067866/204032182-983cf8ff-7534-437d-b24d-26ef5fa2b7f1.png)
![git3](https://user-images.githubusercontent.com/119067866/204032196-cfa6cbfe-d82a-4a47-a9cc-a7495172e851.png)

Maybe I set a wrong learning rate or wrong path of dataset? I found train set and validation set without nn.png. Could you provide me some suggestions? Thanks!

My dataset directory: 
```
/home/kf/nerfactor/data/selected/hotdog_2163
│   transforms_test.json
│   transforms_train.json
│   transforms_val.json
│   
├───test_000
│       albedo.png
│       diffuse-color.exr
│       metadata.json
│       nn.png
│       normal.exr
│       normal.png
│       refball-normal.exr
│       refball-normal.png
│       rgba_city.png
│       rgba_courtyard.png
│       rgba_forest.png
│       rgba_interior.png
│       rgba_night.png
│       rgba_olat-0000-0000.png
│       rgba_olat-0000-0008.png
│       rgba_olat-0000-0016.png
│       rgba_olat-0000-0024.png
│       rgba_olat-0004-0000.png
│       rgba_olat-0004-0008.png
│       rgba_olat-0004-0016.png
│       rgba_olat-0004-0024.png
│       rgba_studio.png
│       rgba_sunrise.png
│       rgba_sunset.png
│       rgba.png
│       
├───test_001
│       .......
├───test_199
│       .......
├───train_000
│       albedo.png
│       diffuse-color.exr
│       metadata.json
│       normal.exr
│       normal.png
│       refball-normal.exr
│       refball-normal.png
│       rgba_city.png
│       rgba_courtyard.png
│       rgba_forest.png
│       rgba_interior.png
│       rgba_night.png
│       rgba_olat-0000-0000.png
│       rgba_olat-0000-0008.png
│       rgba_olat-0000-0016.png
│       rgba_olat-0000-0024.png
│       rgba_olat-0004-0000.png
│       rgba_olat-0004-0008.png
│       rgba_olat-0004-0016.png
│       rgba_olat-0004-0024.png
│       rgba_studio.png
│       rgba_sunrise.png
│       rgba_sunset.png
│       rgba.png
│       
├───train_001

│       
├───train_002
│       metadata.json
│       rgba.png
│       
├───train_002
│       .......
├───train_099
│     
├───val_000
│       albedo.png
│       diffuse-color.exr
│       metadata.json
│       normal.exr
│       normal.png
│       refball-normal.exr
│       refball-normal.png
│       rgba_city.png
│       rgba_courtyard.png
│       rgba_forest.png
│       rgba_interior.png
│       rgba_night.png
│       rgba_olat-0000-0000.png
│       rgba_olat-0000-0008.png
│       rgba_olat-0000-0016.png
│       rgba_olat-0000-0024.png
│       rgba_olat-0004-0000.png
│       rgba_olat-0004-0008.png
│       rgba_olat-0004-0016.png
│       rgba_olat-0004-0024.png
│       rgba_studio.png
│       rgba_sunrise.png
│       rgba_sunset.png
│       rgba.png
│ 
├───val_001
│     ........
└───val_007

```
